src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -57,6 +58,8 @@ export default class SaveInProgressIntro extends React.Component {
           </div>
         );
       }
+    } else if (renderSignInMessage) {
+      alert = renderSignInMessage(prefillEnabled);
```

Comment:
It doesn't look like this prefillEnabled argument is used in this case, is it used elsewhere?

Reply:
It's not used in the VIC page, but since I'm skipping that branch, I figured it would be useful in the future, possibly.